app/show_demo/conf/swing_arms_pid.txt

```diff
@@ -0,0 +1 @@
+12274
```

Comment:
Same comment as above.

Reply:
Ok, that's an error. These are just files created by the bash scripts. I'll remove them immediately.